recipe/meta.yaml

```diff
@@ -1,24 +1,18 @@
-{% set version = "58.5.3" %}
+{% set version = "59.0.1" %}
 
 package:
   name: setuptools
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/s/setuptools/setuptools-{{ version }}.tar.gz
-  sha256: dae6b934a965c8a59d6d230d3867ec408bb95e73bd538ff77e71fedf1eaca729
+  sha256: 899d27ec8104a68d4ba813b1afd66708a1a10e9391e79be92c8c60f9c77d05e5
   patches:
-    # Modify setuptools to fail if used in conda build (encourage people to add all deps in meta.yaml).
-    - patches/0002-disable-downloads-inside-conda-build.patch
```

Comment:
This patch doesn't prevent installing with pip. conda-build sets `PIP_NO_INDEX=1` anyway which should be enough.